core/chains/evm/txm/types/transaction.go

```diff
@@ -0,0 +1,203 @@
+package types
```

Comment:
A couple of reasons include:
- Cleaning up: some things are not needed anymore
- Removing generics: we wanted to degeneralize the TXM anyway so this seems the way to do so
- Cyclic references: I wasn't sure if I would hit any cyclic references errors so I tried to avoid having dependencies to the existing TXM